Makefile

```diff
@@ -34,6 +36,28 @@ install: all
 	ln -rsf ${VKDTDIR}/vkdt $(DESTDIR)$(prefix)/bin/vkdt
 	ln -rsf ${VKDTDIR}/vkdt-cli $(DESTDIR)$(prefix)/bin/vkdt-cli
 
+libinstall: ext lib bin
+	mkdir -p $(VKDTLIBDIR)
```

Comment:
this one is not necessary because `mkdir -p` will make the parent too

Reply:
Agree.